src/sentry/static/sentry/app/utils.tsx

```diff
@@ -91,7 +91,7 @@ export function sortArray<T>(arr: Array<T>, score_fn: (entry: T) => string): Arr
   return arr;
 }
 
-export function objectIsEmpty(obj: object): boolean {
+export function objectIsEmpty(obj?: object): boolean {
```

Comment:
I would not do that... if I am checking an object, I should pass an object... we should verify if the object is undefined before using this method...or you can 

```suggestion
export function objectIsEmpty(obj: {}): boolean {
```